addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3234,9 +3234,20 @@ export class OdooEditor extends EventTarget {
             if (!node.matches || node.matches(CLIPBOARD_BLACKLISTS.remove.join(','))) {
                 node.remove();
             } else {
-                // Unwrap the illegal node's contents.
-                for (const unwrappedNode of unwrapContents(node)) {
-                    this._cleanForPaste(unwrappedNode);
+                let nodes;
```

Comment:
I would rename this to `children` because that's what that variable is going to contain, right?

Reply:
@dmo-odoo how about `childrens` so that i can do `for (const children of childrens) {` in the for loop below. ^^'

Reply:
@deso-odoo But "children" is already the plural. The singular is "child". Don't you mean `for (const child of children) {` ?

Reply:
@dmo-odoo :eyes: I forgot children itself is plural. :grimacing:. Sorry my bad. :face_with_peeking_eye: